### DISCUSSION TITLE
[预览] v2.6.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3900] [分享 / Share]
包含 v2.6.0 的所有更新内容.

✨新增
- 新增功能 `批量导入合集`. (PR #3884 by [swhoro](https://github.com/swhoro))
> 在合集页面制作一个批量导入按钮，可以新增所有合集内视频至同名收藏夹

- `扩展倍速` 添加 `隐藏移除图标` 和 `隐藏新增图标` 选项. (PR #3864 by [Tinhone](https://github.com/Tinhone))
- 新增插件 `快捷键扩展 - 开关 CC 字幕`, 默认为 <kbd>Shift</kbd> + <kbd>C</kbd>. (#3407)

🐛修复
- 修复新版播放页下 `简化评论区` 的样式. (#3872)
- `StreamSaver` 下载补充三方 cookie 说明. (#3824)

☕开发者相关
- 完成 define API + SwitchOptions API 的改造 (受影响组件为 `简化首页` 和 `简化直播间`), 感谢 [timongh](https://github.com/timongh) 的长期贡献.
- 重新拆分了一下 PlayerAgent 的文件.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.0-preview'>[预览] v2.6.0</a>.</em>

**Comment by peace-max** [comment]
最近好多vpn扩展，都用不了不容易安装。大家有什么扩展可以推荐一下吗？
